upup/models/cloudup/resources/addons/karpenter.sh/k8s-1.19.yaml.template

```diff
@@ -929,108 +989,35 @@ spec:
-        operator: Exists
-      topologySpreadConstraints:
-      - maxSkew: 1
-        topologyKey: "topology.kubernetes.io/zone"
-        whenUnsatisfiable: ScheduleAnyway
-        labelSelector:
-          matchLabels:
-            karpenter: webhook
-      - maxSkew: 1
-        topologyKey: "kubernetes.io/hostname"
-        whenUnsatisfiable: DoNotSchedule
-        labelSelector:
-          matchLabels:
-            karpenter: webhook
-      nodeSelector: null
-
----
-# Source: karpenter/templates/webhook/deployment.yaml
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: karpenter-webhook
-  namespace: kube-system
-spec:
-  replicas: {{ ControlPlaneControllerReplicas false }}
-  strategy:
-    type: Recreate
-  selector:
-    matchLabels:
-      karpenter: webhook
-  template:
-    metadata:
-      labels:
-        app.kubernetes.io/name: karpenter
-        karpenter: webhook
-    spec:
-      priorityClassName: system-cluster-critical
-      serviceAccountName: karpenter
-      dnsPolicy: Default
-      containers:
-        - name: webhook
-          image: public.ecr.aws/karpenter/webhook:v0.16.3
-          imagePullPolicy: IfNotPresent
-          args:
-            - -port=8443
-          resources: 
-            limits:
-              cpu: 100m
-              memory: 50Mi
-            requests:
-              cpu: 100m
-              memory: 50Mi
           ports:
+            - name: http-metrics
+              containerPort: 8080
+              protocol: TCP
+            - name: http
+              containerPort: 8081
+              protocol: TCP
             - name: https-webhook
               containerPort: 8443
               protocol: TCP
           livenessProbe:
             initialDelaySeconds: 30
+            timeoutSeconds: 30
             httpGet:
-              scheme: HTTPS
-              port: https-webhook
+              path: /healthz
+              port: http
           readinessProbe:
+            timeoutSeconds: 30
             httpGet:
-              scheme: HTTPS
-              port: https-webhook
-          startupProbe:
-            httpGet:
-              scheme: HTTPS
-              port: https-webhook
-            failureThreshold: 6
-          env:
-            - name: AWS_REGION
-              value: {{ Region }}
-            - name: CLUSTER_NAME
-              value: {{ ClusterName }}
-            - name: KUBERNETES_MIN_VERSION
-              value: "1.19.0-0"
-            - name: CLUSTER_ENDPOINT
-              value: https://{{ APIInternalName }}
-            - name: CONFIG_LOGGING_NAME
-              value: "karpenter-config-logging"
-            - name: KARPENTER_SERVICE
-              value: karpenter-webhook
-            - name: SYSTEM_NAMESPACE
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.namespace
+              path: /readyz
+              port: http
+          resources:
+            limits:
+              cpu: 1
```

Comment:
This seems a bit much. The pod won't fit on the control plane nodes.

Reply:
Resources updated :ok_hand: 
 - https://github.com/kubernetes/kops/pull/15144/commits/322d94d03043c423b72b0a72e821acf73d68a3c4